code/game/supplyshuttle.dm

```diff
@@ -46,6 +46,29 @@ var/list/mechtoys = list(
 	layer = 4
 	explosion_resistance = 5
 
+/obj/structure/plasticflaps/attackby(obj/item/I as obj, mob/user as mob)
+	..()

```

Comment:
This is still in there twice.
You need to remove this, and leave the return ..() at the end.